tools/doc/html.js

```diff
@@ -31,6 +31,7 @@ const typeParser = require('./type-parser.js');
 module.exports = toHTML;
 
 const STABILITY_TEXT_REG_EXP = /(.*:)\s*(\d)([\s\S]*)/;
+const DOC_CREATED_REG_EXP = /<!--introduced_in=v([0-9]+).([0-9]+).([0-9]+)-->/;
```

Comment:
The dots need to be escaped. Also, you can optionally relax the RegEx to accept optional spaces.